assets/app/index.rb

```diff
@@ -5,6 +5,32 @@ def render
     css = <<~CSS
       * { font-family: 'Inconsolata', monospace; }
 
+      @media (prefers-color-scheme: dark) {
+        body {
+          background-color: black;
+          color: #eee;
+        }
+        .nav #logo--dark path {
+          fill: #eee;
+        }
+        .nav #logo--dark polygon {
+          fill: #ec232a;
+        }
+        .nav__links a {
```

Comment:
why is this double under score?

Reply:
BEM naming convention, it’s pretty handy in regards to CSS. https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

Reply:
i see, although we don't use it anywhere else, so i'd prefer to change just to snake case